component/badge/badge.css

```diff
@@ -0,0 +1,76 @@
+.badge {
+    font-size: 0.70em;
+    display: inline-block;
+    text-align: center;
+    border-radius: 0.25rem;
+    padding: 0.35em 0.65em;
+    min-width: 1em;
+}
+
+.btn .badge {
+    position: relative;
+    top: -1px;
+}
+
+.bg-primary {
+    color: var(--white);
+    background-color: var(--primary-color);
+}
+
+.bg-secondary {
+    color: var(--white);
+    background-color: var(--secondary-color);
+}
+
+.bg-danger {
+    color: var(--white);
+    background-color: rgb(255, 67, 67);
```

Comment:
You should always use variables for colours. You can check all of your files in this PR and use variables.

Reply:
Yes thanks I will fix it.